tutorials/30_File_Type_Preprocessing_Index_Pipeline.ipynb

```diff
@@ -0,0 +1,1328 @@
+{
```

Comment:
<p>Unfortunately, this method is not working when we convert this ipynb into a python script and run it in nightly test. You need to update this section with this code:</p><pre class="ql-syntax" spellcheck="false">import os
from getpass import getpass

api_key = os.getenv("OPENAI_API_KEY", None) or getpass("Enter OpenAI API key:")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/274/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
oh dang that's too bad.  I thought I was in the clear because the build passed. Will update!